tests/test_main.py

```diff
@@ -24,12 +27,39 @@ def test_generate_fails_with_existing_yml():
     """Test that ensures that a second yml file isn't generated
     without the force command."""
     result = runner.invoke(main.app, ["generate"])
+    print(result.stdout)
 
     assert result.exit_code == 0
 
 
 def test_generate_force_option_creates_yml():
     """Test that ensures the force command works correctly."""
     result = runner.invoke(main.app, ["generate"])
+    print(result.stdout)
+
+    assert result.exit_code == 0
+
+
+@pytest.mark.parametrize(
+    "assignment_path,expected_checks",
```

Comment:
Since the second parameter is a list of strings we want in our output, might it be better to name the second parameter something like `expected_outputs`?